src/integrations/stytch.js

```diff
@@ -0,0 +1,85 @@
+import { getStytchJwt } from '../utils/browser';
+import { getAuthClient, setAuthClient } from '../utils/init';
+import { dynamicImport } from './utils';
+
+let stytchPromise = null;
+let config = null;
+// This guard prevents a second auth attempt to stytch if the form is preloaded
+let authSent = false;
+
+export function installStytch(stytchConfig) {
+  if (stytchPromise) return stytchPromise;
+  else if (!stytchConfig) return Promise.resolve();
+  else {
+    config = stytchConfig;
+    stytchPromise = new Promise((resolve) => {
+      const stytchClient = getAuthClient();
+      if (stytchClient) resolve(stytchClient);
+      else {
+        // Bring in stytch dependencies dynamically if this form uses stytch
+        return dynamicImport(['https://js.stytch.com/stytch.js'], false).then(
+          () => {
+            const initializedClient = global.Stytch(stytchConfig.api_key);
+            setAuthClient(initializedClient);
+            resolve(initializedClient);
+          }
+        );
+      }
+    });
+    return stytchPromise;
+  }
+}
+
+export async function sendMagicLink(fieldVal) {
+  getAuthClient().magicLinks.email.loginOrCreate(fieldVal, {
```

Comment:
should this be returned? even if the return value isn't being used right now

Reply:
yeah, good point